src/commands.ts

```diff
@@ -589,6 +594,21 @@ export class Commands {
 			}
 			workspaceOwner = workspace.owner_name;
 			workspaceName = workspace.name;
+		} else {
+			workspaceOwner = args[0] as string;
+			workspaceName = args[1] as string;
+			workspaceAgent = args[2] as string | undefined;
+			folderPath = args[3] as string | undefined;
+			openRecent = args[4] as boolean | undefined;
+		}
+
+		if (workspaceAgent === undefined) {
```

Comment:
I think maybe we want `if (!workspaceAgent)` in case someone has `agent=` in the URI (so a blank string).